styles/age.css

```diff
@@ -1,35 +1,67 @@
+@font-face {
+    font-family: 'Retro';
+    src: url('/assets/font/retro_computer_personal_use.ttf'); /*URL to font*/
+}
 body{
     margin: 0;
     padding: 0;
+    color: white;
+    font-family: Retro; text-shadow: 2px 2px rgba(255, 255, 255, 0.5);
+    font-size: 17px;
+    letter-spacing: 1px;
+    line-height: 28px;
 }
-
-#hero{
+#hero {
     height:100vh;
     width:100%;
     display:flex;
-    align-items: center;
+    align-items:center;
     justify-content: center;
-    background-image:linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),    url("../agecheck.gif");
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
     position: relative;
-    color: white;
+    overflow: hidden;
+    background:rgba(0, 0, 0, 0.75);
+}
+#bgvideo {
+    position: absolute;
+    top: 60%;
+    left: 50%;    
+    width: auto;
+    height: auto;
+    z-index: -1;
+    -webkit-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    background-size: contain;
 }
 #box{
-    width: 250px;
+    z-index: 100;
+    width: 2500px;
     text-align: center;
-    background-color: rgba(0, 0, 0, 0.75);
-    padding: 20px;
+    background-color: rgba(0, 0, 0, 0.5);
+    padding: 50px;
 }
-#submit{
-    text-align: center; margin-top: 10px;
-}
-#input{
-    height: 35px; max-width: 25%;
-    margin: 0 5px;
+#submit-button{
+    font-family: Retro;
+    text-align: center;
+    margin-top: 0px;
```

Comment:
No effect. If your use Margin-top: 0px and after use margin property margin-top will be overwritten.

Delete this line